awsiot/mqtt_connection_builder.py

```diff
@@ -39,6 +39,30 @@
 
             *   `**kwargs` (dict): Forward-compatibility kwargs.
 
+    **on_connection_success** (`Callable`): Optional callback invoked whenever the connection successfully connects.
+        The MQTT client will automatically attempt to reconnect.
```

Comment:
Actually, I think we should remove the automatically attempt to reconnect comment on all of these three.